packages/fiori/test/pages/ShellBar.html

```diff
@@ -240,7 +240,7 @@ <h3>ShellBar in Compact</h3>
 	</ui5-shellbar>
 
 	<script>
-		mySearch.addEventListener("ui5-suggestion-item-select", function (event) {
+		mySearch.addEventListener("ui5-selection-change", function (event) {
```

Comment:
would listening for selection change be sufficient here? Before the suggestion-item-select is was fired only on final item selection

Reply:
I left this because the only thing this function does is logging information in the console and I did not see any tests or functionality related to it.